content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
```

Comment:
I quite like "jEdit bugs: Getting off the Ground with Kinto and Kinto.js" if it's OK with you.